File: app/src/main/java/com/github/kongpf8848/animation/activity/TransitionActivity.java
package com.github.kongpf8848.animation.activity;

import android.view.View;

import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.transition.ShareElementOneActivity;

import butterknife.OnClick;

public class TransitionActivity extends BaseToolbarActivity {

    @Override
    protected int getLayoutId() {
        return R.layout.activity_transition;
    }

    @OnClick(R.id.button1)
    public void onButton1(View view) {
        startActivity(ShareElementOneActivity.class);
    }


}

File: app/src/main/java/com/github/kongpf8848/animation/activity/TransitionActivity.kt
package com.github.kongpf8848.animation.activity

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.activity.transition.ShareElementOneActivity

class TransitionActivity : BaseToolbarActivity() {

    override fun getLayoutId(): Int {
        return R.layout.activity_transition
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?) {
        startActivity(ShareElementOneActivity::class.java)
    }
}